proposals/NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md

```diff
@@ -0,0 +1,75 @@
+# Handling VR help requests when application does not send VR help prompt
+
+* Proposal: [SDL-NNNN](NNNN-New_rules_for_providing_VRHelpItems_VRHelpTitle.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: Core, RPC
+
+## Introduction
+
+"vrHelp" items and "helpPrompt" are set by SetGlobalProperties for helping user to make voice navigation over registered application, what is very useful in case your hands are busy with driving.
+Safety requirements are very strong in automotive, so possibility to use Voice Recognition should be provided for driver all the time.
+
+## Motivation
+
+If there is no SetGlobalProperties issued, the default help item list will be populated by command list shared by Voice module or by default values from .ini file. If application issues a SetGlobalProperties with help item list, then this list will have precedence and it will be displayed on HMI.
+But there is no guarantee that Application provide "vrHelp" and "helpPrompt", and possible situation when driver couldn't use Voice Recognition.
+The reason of proposal to add logic of adding values for "vrHelp" item and "helpPrompt" in case they are not provided by application.
+
+## Proposed solution
+
+The solution proposed here is to implement next items:
+1. In case mobile application does NOT send SetGlobalProperties request with "vrHelp" and "helpPrompt" at all during 10 sec timer and this mobile application has NO registered AddCommand and/or DeleteCommand requests (resumed during data resumption).
+SDL must provide the default values of "helpPrompt" and "vrHelp" to HMI.
+SDL must start this 10 sec timer only once in case of application registration OR after HMILevel resumption OR activation after registration.
+2. In case mobile application successfully registers and gets any HMILevel other than NONE OR changes it's level from NONE to any other (in case of HMILevel resumption or activation).
+SDL must:
```

Comment:
@GetmanetsIrina 
Please use list formatting option e.g.
- first item
- second item
- third item

since the final document view is almost unreadable

Reply:
:+1: